src/client/components/Pack.tsx

```diff
@@ -51,10 +50,6 @@ const Pack: React.FC<PackProps> = ({
           <Button onClick={onRetry} color="danger" disabled={retryInProgress}>
             {retryInProgress ? 'Retrying...' : 'Bot picks failed. Try again?'}
           </Button>
-        ) : loadingPredictions ? (
-          <Button color="secondary" disabled>
-            Making Bot Picks...
-          </Button>
         ) : (
           ratings &&
           ratings.length > 0 &&
```

Comment:
Minor UX thing but I noticed that when the button isn't showing (ratings loading or showing the ratings) the height of the header changes, as seen in gif:
![header-height-shifting](https://github.com/user-attachments/assets/57970f98-267d-4668-b80e-466e7647a182)
My suggested solution is to move the conditional onto the button to set class invisible eg `className={ratings && ratings.length > 0 && !showRatings ? '' : 'invisible'}`. That way the button is always in the layout for height calculations even if not visible currently.